sig/handwritten/app/controllers/users_controller.rbs

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  @user: User
+
+  @profile: Profile
+
+  # TODO: In implementation, `@token` is surely an optional string. But steep does not support type-narrowing for
+  #       instance variables.  Therefore it will cause type errors to give a type to this variable.
+  # @token: String?
```

Comment:
At present, there is no way to give types for this.
We can avoid this limitation by not using instance variables, but it's a bit ugly...